TwitchChannelPointsMiner/classes/AnalyticsServer.py

```diff
@@ -20,17 +20,21 @@ def streamers_available():
     ]
 
 
-def read_json(streamer):
+def read_json(streamer, baseurl):
     path = Settings.analytics_path
     streamer = streamer if streamer.endswith(".json") else f"{streamer}.json"
+    streamer_without_baseurl = streamer.replace(baseurl, "")
```

Comment:
To me this is dangerous.

What if your base url is `banana` and you got a streamer whose name is `ilovebananas`?

Maybe a better solution would be to remove the first `n` (length of `baseurl`) characters. Or set a maximum replacement amount `streamer.replace(baseurl, "", 1)`.

Reply:
It could be removed